packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -129,10 +129,10 @@ export async function fetchHelper(
             online = OnlineStatus.Offline;
         }
         if (error.name === "AbortError") {
-            throwOdspNetworkError("timeoutDuringFetch", fetchTimeoutStatusCode);
+            throw new RetryableError("fetchAbort", "Fetch Timeout (AbortError)", OdspErrorType.fetchTimeout);
```

Comment:
Is this the right error type?  Can we get a comment explaining which subsystem creates "AbortError"s and what it means specifically?  Same with the one below (it's obviously timeout but still some context would be good)

Reply:
There is no change in type here - same OdspErrorType.fetchTimeout used as before.
fetch() returns these errors and official documentation on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) is not very dense.

That said, I believe this error is due to cancelation (i.e. using abort signal) only.

Reply:
Added short comment on when these errors are thrown.
Also opened https://github.com/microsoft/FluidFramework/issues/8488 to introduce and use new error type here